mmdet/evaluation/metrics/crowdhuman_metric.py

```diff
@@ -0,0 +1,814 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import copy
+import json
+import os.path as osp
+import tempfile
+from collections import OrderedDict
+from multiprocessing import Process, Queue
+from typing import Dict, List, Optional, Sequence, Union
+
+import numpy as np
+from mmengine.evaluator import BaseMetric
+from mmengine.logging import MMLogger
+from scipy.sparse import csr_matrix
```

Comment:
scipy is not the necessary package of  MMDetection. Need to handle the import issue.
```suggestion
try:
    from scipy.optimize import csr_matrix, maximum_bipartite_matching
except ImportError:
    csr_matrix,  maximum_bipartite_matching = None, None
```

and raise error during metric class init like this:
```
        if csr_matrix is None:
            raise ImportError('Please run "pip install scipy" '
                              'to install scipy first.')
```